src/components/new-members/index.js

```diff
@@ -5,19 +5,46 @@ import styles from '@components/new-members/new-members.module.scss';
 import { membersContext } from '@store/members/members-context';
 import { searchMemberContext } from '@store/search-members/searchMembers-context';
 import { searchMembers } from '@helper-functions/search-members';
+import Link from 'next/link';
+import { userContext } from '@store/user/user-context';
 
 // returns card which shows details of new member
-const renderNewMember = (newMember, isOptionKey) => (
-  <div className={styles.containerForNewMember}>
-    <Card
-      developerInfo={newMember}
-      isMember={false}
-      isOptionKey={isOptionKey}
-    />
-  </div>
-);
+const renderNewUserCard = (newMember, isOptionKey) => {
+  return (
+    <div className={styles.containerForNewMember}>
+      <Card
+        developerInfo={newMember}
+        isMember={false}
+        isOptionKey={isOptionKey}
+      />
+    </div>
+  );
+};
+
+const renderNewUsers = (newMember, isOptionKey, isSuperUser) => {
+  if (isSuperUser && isOptionKey) {
+    return (
+      <Link
+        prefetch={false}
+        href={{
+          pathname: '/[id]',
+          query: {
```

Comment:
Why is this query needed?

Reply:
yeah, this is not needed. I will remove it.